README.md

```diff
@@ -63,7 +63,7 @@ Steps:
 - Add a configuration property inside you `application.properties` file: `greeting=World` 
 - Modify `BookmarkResource` to read this configuration property and log it at startup:
   - Use `@ConfigProperty` to inject the property inside a `greeting` variable.
-  - Create a `@PostConstruct` method that log it via `log.info("Hello {}", greeting)` using [SLFJ](https://www.slf4j.org/).
+  - Create a `@PostConstruct` method that log it via `LOGGER.infof("Hello %s", greeting)` using [Jboss logging](https://docs.jboss.org/jbosslogging/latest/org/jboss/logging/Logger.html).
```

Comment:
See my other comment, please don't do this if it previously works as the LOG usage is froma different step